### PR TITLE
feat(dashboard): name edit modal in detail-page markups

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA
- Waiting for API
- Most of codes are copied from 'CostDashboardUpdateModal' 

스크린샷 2022-12-12 오후 3 38 32

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            setForm,
+            initForm,
+            invalidState,
+            invalidTexts,
+            validate,
+            isAllValid,
+        } = useFormValidator({
+            name: '',
+        }, {
+            name(value: string) { return value.trim().length ? '' : 'Required Field'; },
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue:86]: 🧐
```suggestion
            name(value: string) { return value.trim().length ? '' : 'Required Field'; }, // song-lang
```

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue (hunk 1 above)
Comment (minseolee) [reply src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue:86]: GOOD!!

Comment (WANZARGEN) [inline src/services/dashboards/dashboard-detail/modules/DashboardNameEditModal.vue]: issue: Validation result also needs i18n :)

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**